client_cli/pytests/common/settings.py

```diff
@@ -11,11 +11,9 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-ROOT_DIR = os.environ.get("CLIENT_CLI_DIR", BASE_DIR)
-
 PATH_CONFIG_CLIENT_CLI = os.environ["CLIENT_CLI_CONFIG"]
 CLIENT_CLI_PATH = os.environ["CLIENT_CLI_BINARY"]
-PEERS_CONFIGS_PATH = os.path.join(ROOT_DIR, "peers_configs")
+PEERS_CONFIGS_PATH = os.path.join(BASE_DIR, "peers_configs")
```

Comment:
What this is for? I found it was introduced by #4036, but there is no documentation for it (please add to README).

Reply:
removed it